README.md

```diff
@@ -44,4 +44,15 @@ The datastore can be configured with the following environment variables:
 - `DATASTORE_MIN_CONNECTIONS`: The minimum amount of connections to the database that will be created. Default: 0
 - `DATASTORE_MAX_CONNECTIONS`: The maximum amount of connections to the database that will be created. If this is set to 1, only one connection can access the database at a time. The writer always runs in single-access mode, so no write errors occur. Default: 1
 - `DATASTORE_MAX_RETRIES`: The amount of times a request to the database is retried before giving up. Minimum: 1, Default: 3
-- `DATASTORE_RETRY_TIMEOUT`: How long to wait before retrying a request to the database, in ms. Set 0 to disable waiting between requests. Default: 10
+- `DATASTORE_RETRY_TIMEOUT`: How long to wait before retrying a request to the database, in ms. Set 0 to disable waiting
+  between requests. Default: 10
+- `OPENSLIDES_DEVELOPMENT`: If set to a truthy value, the datastore runs in development mode (see [development docs](docs/development.md)
+  for the implications).
+- `DATASTORE_LOG_LEVEL`: Set the log level for the datastore. If not provided, it defaults to `DEBUG` in development
+  mode and `INFO` otherwise.
+- `DATASTORE_DATABASE_HOST`: Host of the underlying database. Default: `postgresql`
+- `DATASTORE_DATABASE_PORT`: Port of the underlying database. Default: `5432`
+- `DATASTORE_DATABASE_USER`: User to use to login to the database. Default: `openslides`
```

Comment:
We also have `DATASTORE_DATABASE_NAME` which defaults to `openslides`. Do you use this? See https://github.com/OpenSlides/openslides-manage-service/blob/main/pkg/config/default-docker-compose.yml#L15

Maybe just add it here.

Reply:
Yes, thanks for the hint, seems like I missed this